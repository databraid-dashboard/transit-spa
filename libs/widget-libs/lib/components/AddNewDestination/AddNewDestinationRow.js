var _jsxFileName = '../../src/components/AddNewDestination/AddNewDestinationRow.jsx',
    _this = this;

import React from 'react';
import AddNewDestinationModal from './AddNewDestinationModal';

var AddNewDestinationRow = function AddNewDestinationRow() {
  return React.createElement(AddNewDestinationModal, {
    __source: {
      fileName: _jsxFileName,
      lineNumber: 4
    },
    __self: _this
  });
};

export default AddNewDestinationRow;