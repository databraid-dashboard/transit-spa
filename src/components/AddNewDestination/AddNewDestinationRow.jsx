import React from 'react';
import AddNewDestinationModal from './AddNewDestinationModal';

const AddNewDestinationRow = () => <AddNewDestinationModal />;

export default AddNewDestinationRow;
