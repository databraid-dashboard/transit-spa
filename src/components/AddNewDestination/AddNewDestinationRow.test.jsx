import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AddNewDestinationRow from './AddNewDestinationRow';

describe('AddNewDestinationRow', () => {
  it('should render row', () => {
    const component = shallow(<AddNewDestinationRow />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
