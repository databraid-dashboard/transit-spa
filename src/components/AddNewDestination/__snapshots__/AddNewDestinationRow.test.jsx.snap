// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddNewDestinationRow should render row 1`] = `<AddNewDestinationModal />`;
